### ISSUE TITLE
Skin Editor Crash

### ISSUE THREAD

**Issue by SodiumShine** [tom5454/CustomPlayerModels#5]
Crashes when the main menu button "Open Skin Editor" is clicked. The whole game doesn't crash, just the editor. Returns to main menu after.
MC 1.16.5
Forge 36.0.14
Custom Player Models 1.16 v0.0.6a

```
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: java.lang.NullPointerException
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.entity.LivingEntity.<init>(LivingEntity.java:205)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.entity.player.PlayerEntity.<init>(PlayerEntity.java:159)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.entity.player.AbstractClientPlayerEntity.<init>(AbstractClientPlayerEntity.java:33)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.entity.player.RemoteClientPlayerEntity.<init>(RemoteClientPlayerEntity.java:16)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.client.FakePlayer.<init>(FakePlayer.java:18)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.client.FakePlayer.<init>(FakePlayer.java:22)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.client.ViewportPanelImpl.<init>(ViewportPanelImpl.java:66)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.shared.gui.NativeGuiComponents.getNative(NativeGuiComponents.java:11)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.shared.editor.gui.ViewportPanel.<init>(ViewportPanel.java:20)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.shared.editor.gui.EditorGui.initModelPanel(EditorGui.java:92)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.shared.editor.gui.EditorGui.initFrame(EditorGui.java:68)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.shared.gui.Frame.init(Frame.java:30)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.client.GuiImpl.func_231160_c_(GuiImpl.java:137)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.gui.screen.Screen.func_231158_b_(Screen.java:325)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.Minecraft.func_147108_a(Minecraft.java:852)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.client.ClientProxy.lambda$openGui$0(ClientProxy.java:94)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.tom.cpm.client.ClientProxy$Button.lambda$new$0(ClientProxy.java:143)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.gui.widget.button.Button.func_230930_b_(SourceFile:33)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.gui.widget.button.AbstractButton.func_230982_a_(SourceFile:16)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.gui.widget.Widget.func_231044_a_(Widget.java:136)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.gui.INestedGuiEventHandler.func_231044_a_(SourceFile:27)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.gui.screen.MainMenuScreen.func_231044_a_(MainMenuScreen.java:288)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.MouseHelper.lambda$mouseButtonCallback$0(MouseHelper.java:87)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.gui.screen.Screen.func_231153_a_(Screen.java:427)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.MouseHelper.func_198023_a(MouseHelper.java:85)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.MouseHelper.lambda$null$4(MouseHelper.java:175)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.util.concurrent.ThreadTaskExecutor.execute(SourceFile:94)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.MouseHelper.lambda$registerCallbacks$5(MouseHelper.java:174)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at org.lwjgl.system.JNI.invokeV(Native Method)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(SourceFile:102)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.MainWindow.func_227802_e_(MainWindow.java:305)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:996)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraft.client.main.Main.main(Main.java:184)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82)
[12:28:30] [Render thread/INFO]: [com.tom.cpm.client.GuiImpl:func_231160_c_:139]: 	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)
```
Full Log:
[latest.log](https://github.com/tom5454/CustomPlayerModels/files/5938005/latest.log)

**Comment by agentwolf88** [comment]
Same

https://paste.ee/p/BPxI6

**Comment by tom5454** [comment]
I released a new version 0.0.6b that fixes this bug.

**Comment by agentwolf88** [comment]
Still crashs

**Comment by tom5454** [comment]
Can you post your log?

**Comment by agentwolf88** [comment]
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: java.lang.NullPointerException
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.github.ipixeli.gender.client.MCAccessor.getPlayers(MCAccessor.java:104)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.github.ipixeli.gender.client.MCAccessor.getPlayerByUuid(MCAccessor.java:116)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.github.ipixeli.gender.client.Renders.onEvent(Renders.java:84)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at net.minecraft.client.resources.DefaultPlayerSkin.func_177334_a(SourceFile)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.tom.cpm.client.ViewportPanelImpl.render(ViewportPanelImpl.java:125)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.tom.cpm.shared.editor.gui.ViewportPanel.draw(ViewportPanel.java:29)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.tom.cpm.shared.gui.elements.Panel.draw(Panel.java:29)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.tom.cpm.shared.gui.elements.Panel.draw(Panel.java:29)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.tom.cpm.shared.gui.elements.Panel.draw(Panel.java:29)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at com.tom.cpm.client.GuiImpl.func_230430_a_(GuiImpl.java:95)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:290)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at sun.reflect.GeneratedMethodAccessor26.invoke(Unknown Source)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at net.optifine.reflect.Reflector.callVoid(Reflector.java:723)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:821)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:976)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at net.minecraft.client.main.Main.main(Main.java:184)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:50)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
[13:59:29] [Render thread/INFO] [STDERR/]: [com.tom.cpm.client.GuiImpl:func_230430_a_:97]: 	at org.multimc.EntryPoint.main(EntryPoint.java:34)


https://paste.ee/p/JBIYq

**Comment by tom5454** [comment]
The crash is in the Gender Mod, I uploaded a new version to fix this (0.0.6c).
